src/Native/Silk.NET.Assimp.Native/Silk.NET.Assimp.Native.csproj

```diff
@@ -17,20 +13,6 @@
     <Description>Multi-platform native library for Assimp.</Description>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/assimp/assimp</RepositoryUrl>
-    <IncludeBuildOutput>false</IncludeBuildOutput>
-    <IncludeContentInPack>true</IncludeContentInPack>
```

Comment:
Has this moved somewhere else? Looks like it's still having the effect we want (looking at the latest artifacts) but I must've missed something...